utilities/_border.scss

```diff
@@ -53,30 +53,28 @@
 
 @if $u-classes-border == true {
 
-  $border-values: none;
+  @each $border-width, $border-width-value in $border-width-values {
```

Comment:
This is generating the utility classes by looping through a list of values.

It's more complex than other loops we have because for this utility we're using aliases like `thin` and `thick` rather than the pure CSS values. You can see how much simpler the loop in [`_float.scss`](https://github.com/fac/fa-css-utilities/blob/master/utilities/_float.scss#L6) is because we don't need/want aliases for its values; `left`, `right`, and `none` are already human-friendly enough.

This method uses [SASS maps](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#maps) to generate the classes by referencing the aliases in `[_utility-aliases.scss](https://github.com/fac/fa-css-utilities/blob/master/_utility-aliases.scss)`